packages/core/src/api/interactions.ts

```diff
@@ -238,14 +444,29 @@ export class InteractionsAPI {
 	public async createModal(
 		interactionId: Snowflake,
 		interactionToken: string,
-		callbackData: APIModalInteractionResponseCallbackData,
+		body: APIModalInteractionResponseCallbackData & RESTPostAPIInteractionCallbackQuery & { with_response?: false },
```

Comment:
Either revert the parameter name change or update the other overloads and TSDocs to match. Perhaps use the same name for this parameter in all methods too